libraries/CAN_comms/Src/CAN_comms.c

```diff
@@ -172,17 +182,18 @@ void CAN_comms_Rx_task(void* argument)
     for(;;)
     {
         /* Wait until there is a message in the queue */ 
-        CAN_comms_Rx_msg_t CAN_comms_Rx_msg;
-        if (osOK == osMessageQueueGet(CAN_comms_Rx_queue, &CAN_comms_Rx_msg, NULL, osWaitForever))
-        {
-            /* Call the handle function pointer */
-            CAN_comms_config.CAN_comms_Rx_callback(&CAN_comms_Rx_msg);
-        }
+    	CAN_comms_Rx_msg_t CAN_comms_Rx_msg;
+    	if (osOK == osMessageQueueGet(CAN_comms_Rx_queue, &CAN_comms_Rx_msg, NULL, osWaitForever))
+    	{
+
+		/* Call the handle function pointer */
+		CAN_comms_config.CAN_comms_Rx_callback(&CAN_comms_Rx_msg);
+    	}
         else
```

Comment:
indent the content inside the `if` statement
![image](https://github.com/user-attachments/assets/44308ad3-5b80-4c64-ae97-3f9690e66d33)